src/app/admin/modules/program-management/admin-programs/components/admin-program-form/admin-program-form.component.ts

```diff
@@ -64,11 +67,15 @@ export class AdminProgramFormComponent implements OnInit {
 
   onSubmit() {
     const req: Program = {
+      id: this.isUpdateMode ? this.program.id : undefined,
```

Comment:
Should disable submit button if `isUpdateMode` is true and `this.program` isn't assigned.